assets/css/input.css

```diff
@@ -705,6 +714,26 @@ section.post {
     margin-bottom: 1rem
 }
 
+
+.page a {
+    text-decoration: underline;
+  outline-color: transparent;
```

Comment:
Vím že žádný code-style guide asi nevedeme a třeba v app.css jsou všude dvě mezery. Ale v input.css jsou všude jinde čtyři mezery, tak by to chtělo i tady.